### ISSUE TITLE
entropy calculation

### ISSUE THREAD

**Issue by bernstei** [libAtoms/pymatnext#6]
confirm that entropy calculation is OK, as recently discussed in https://gaborcsanyi.slack.com/archives/C53HKH341/p1677809789313149  for original pymatnest analysis scripts.

**Comment by bernstei** [comment]
@liviabp I can't find any trace of a PR or commit that fixes the entropy.  Can you remind me what we ended up doing about the issue you noticed?

**Comment by liviabp** [comment]
The link to the slack archive doesn't work for me, so can you remind me which discussion you refer to? (Or do you mean my problems with the total free energy about a month ago?)

**Comment by bernstei** [comment]
Weird about slack.  Yes, I think it's that one.

<img width="994" alt="Screen Shot 2023-04-11 at 7 00 10 PM" src="https://user-images.githubusercontent.com/1202357/231306681-8e6fedb8-e77f-49db-8984-9c2e018474ae.png">

But I know we went back and forth a few times, and I couldn't remember what we settled on as the correct fix.

**Comment by liviabp** [comment]
I think we haven't settled on a fix yet... I was not entirely convinced about all the terms in ns_analyse, but some of my initial arguments were wrong, so I promised to do my derivation again and see if I find the issue (either in the code or in my thinking). I'll share the overleaf with you where I have it started it.

**Comment by bernstei** [comment]
I created #8 to make the new code be like the fixed old code I posted to the slack.  I'm not sure what I was thinking before, but it appears that if I just use `F = log(Z) + E_min`, making sure that `Z` is correctly unshifted, F(T->0) = U(T=0). I'm honestly not sure why I went to the trouble of the previous code. As I said in the overleaf, I think I was worried about the possibility of constant factor in Z, and it was an (incorrect) attempt to account for it.
